site/docs/components/tile.mdx

```diff
@@ -21,39 +21,54 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
-    *   **Supplementary footer** (optional)
-        *   Positioned at bottom of Tile.
-
-*   Anatomy of MultiActionTile
-    *   **Title (required)**
-        *   1-3 words
-    *   **Metadata **(optional)
-        *   Key information to surface that helps people decide between items
-    *   **Information** (optional)
-        *   Object with three paramters: text, primaryAction (optional), secondaryAction (optional)
-        *   Causes Information IconButton to appear in upper right corner. When clicked the backside of Tile is revealed.
-        *   Text params is rendered a paragraph, primaryAction and secondaryAction action rendered as buttons.
-    *   **Children **(optional)
-        *   Placed in middle of Tile between footer and title content.
-    *   **Supplementary footer** (optional)
-        *   **Call to action default button (required):**
-            *   Consider if the label needs to be different across items, especially for people using screen readers. If the labels are the same, they might also need ARIA tags to indicate which card title the button is labeled by or description it’s described by.
-        *   **Secondary control action button **(optional):
-            *   Aim for 1 word only
-*   **Layout**
-    *   Max. 4 per row in 1392px area
-    *   See [Tile Grid](/components/tile-grid)
+### SingleActionTile (not built)
+*   Entire tile is a single click zone for performing a single action.   
+
+### MultiActionTile 
+*   Similar in terms of content to SingleActionTile, but with multiple actions and click zones.  
+*   Supports default action, secondary action and flip action (which shows additional context and optional secondary action).
+
+### InformationTile
+*   For display of hero data. 
+*   Generally has either no action, or a single action to flip (which shows additional context and optional secondary action).
+
+### Layout
+*   Max. 4 per row in 1392px area
+*   See [Tile Grid](/components/tile-grid)
+
+## Anatomy
+
+### MultiActionTile
+*   **Title (required)**
+    *   1-3 words
+*   **Metadata** (optional)
+    *   Key information to surface that helps people decide between items
+*   **Information** (optional)
+    *   Object with three parameters: text, primaryAction (optional), secondaryAction (optional)
+    *   Causes Information IconButton to appear in upper right corner. When clicked the backside of Tile is revealed.
+    *   Text params is rendered a paragraph, primaryAction and secondaryAction action rendered as buttons.
+*   **Children** (optional)
+    *   Placed in middle of Tile between footer and title content.
+*   **Supplementary footer** (optional)
+    *   **Call to action default button (required):**
+        *   Consider if the label needs to be different across items, especially for people using screen readers. If the labels are the same, they might also need ARIA tags to indicate which card title the button is labeled by or description it’s described by.
```

Comment:
❤️

Reply:
Haha that was 100% @ellisjas – all kudo's should be directed to her

Reply:
Kudos @ellisjas !